drivers/mailbox/bcm2836/init.c

```diff
@@ -102,6 +102,9 @@ Return Value:
     NTSTATUS value
 
 --*/
+
+WCHAR macAddrStrGlobal[13] = { 0 };
+
 _Use_decl_annotations_
 NTSTATUS RpiSetDeviceMacAddress (
```

Comment:
Is there a benefit to leaving the existing flow in? I don't know of any conflicts with this registry key, but if the new system is reliable I don't see a need to write the MAC twice.

Reply:
Thank you for your advice. As devices before 3B+ which is registry entry 0001 have different name, LAN9512/LAN9514 USB 2.0 to Ethernet 10/100 Adapter, I need to make a code change to add this device name into the driver and service.